.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - '**.swift'
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  
+  tests:
+    runs-on: self-hosted
+    # runs-on: ubuntu-latest
+    # strategy:
+    #   matrix:
+    #     image:
+    #       - 'swift:5.9'
```

Comment:
5.10?

Reply:
Added both 5.9 and 5.10 to the test matrix + running self-hosted macOS tests.